lib/identity_doc_auth/acuant/responses/get_results_response.rb

```diff
@@ -94,6 +125,47 @@ def attention_with_barcode?
                alert['Disposition'] == BARCODE_COULD_NOT_BE_READ_ERROR)
           end
         end
+
+        def get_image_side_name(side_number)
+          side_number == 0 ? 'front' : 'back'
+        end
+
+        def get_image_info(image_id)
+          @images_by_id ||= raw_images_data.index_by { |image| image["Id"] }
+
+          @images_by_id[image_id]
+        end
+
+        def get_region_info(region_ids)
+          region = regions_by_id[region_ids.first]
+          image = get_image_info(region['ImageReference'])
+
+          {
+            region: region['Key'],
+            side: get_image_side_name(image['Side']),
+          }
+        end
+
+        def process_raw_alerts(alerts)
+          processed_alerts = { passed: [], failed: [] }
+          alerts.each do |raw_alert|
+            region_refs = raw_alert['RegionReferences']
+            new_alert = {
+              name: raw_alert['Key'],
+              result: IdentityDocAuth::Acuant::ResultCodes.from_int(raw_alert['Result']).name
+            }
+
+            new_alert.merge!(get_region_info(region_refs)) if region_refs.present?
+
+            if new_alert[:result] != IdentityDocAuth::Acuant::ResultCodes::PASSED.name
```

Comment:
seems weird to me to parse the result code and throw it away? WDYT of:

```suggestion
            result_code = IdentityDocAuth::Acuant::ResultCodes.from_int(raw_alert['Result'])
            new_alert = {
              name: raw_alert['Key'],
              result: result_code.name
            }

            new_alert.merge!(get_region_info(region_refs)) if region_refs.present?

            if result_code != IdentityDocAuth::Acuant::ResultCodes::PASSED
```

Reply:
Yeah, thanks. That's cleaner.